service/src/test/java/org/cbioportal/genome_nexus/service/internal/VariantAnnotationServiceTest.java

```diff
@@ -42,6 +44,9 @@
     @Mock
     private CancerHotspotServiceImpl cancerHotspotService;
 
+    @Spy
+    private NotationConverter notationConverter;
```

Comment:
using spy decorator to include actual `@Component` utility class, thanks @onursumer